client/components/RFCCard.vue

```diff
@@ -41,7 +41,9 @@
         >
           Abstract
         </Heading>
-        <p class="leading-snug text-gray-800 dark:text-gray-300 pb-2">
```

Comment:
Was this an intentional style change? (Up to you if you care one way or the other, just seems like strange line breaks and doesn't agree with other similar spots)

Reply:
I think because the attribute value got longer then Prettier decide to add some linebreaks. It does look weird but I guess they have to wrap at some length

Reply:
🤮 but I guess that's the price of standardization